woocommerce/woocommerce
# Add WooCommerce header to WooCommerce.com requests

### All Submissions:






### Changes proposed in this Pull Request:


We want to add WooCommerce `user-agent` headers to WooCommerce.com API requests.

### How to test the changes in this Pull Request:


#### Test Extensions

1. Open WooCommerce > Extensions
2. Confirm that Featured extensions, categories, and search are working

#### Test My subscriptions
1. Open WooCommerce > Extensions
2. Open the _My Subscriptions _ tab
3. Connect to WooCommerce.com
4. Update your list of subscriptions
5. Confirm that your subscriptions are loaded

#### Test Status report
1. Open WooCommerce > Status
2. Scroll down to _Remote get_
3. Confirm that the field is loaded

#### Test inbox
1. Delete inbox notifications `DELETE FROM wp_customer_wc_admin_notes`
2. Delete the `wc_remote_inbox_notifications_specs wc_remote_inbox_notifications_stored_state` option
3. Run the `wc_admin_daily` cron: `wp cron event run wc_admin_daily`
4. Open [the home screen](https://wp.test:4443/wp-admin/admin.php?page=wc-admin)
6. Confirm that notifications are loaded

#### Test onboarding
1. Start the onboarding flow
2. On the first step chekc the Mailchimp signup
3. Confirm that all steps are loading correctly and that you can complete the onboarding wizard 


### Other information:




### FOR PR REVIEWER ONLY:
